src/auth/ha-auth-flow.ts

```diff
@@ -98,16 +146,22 @@ class HaAuthFlow extends litLocalizeLiteMixin(LitElement) {
         `;
       case "error":
         return html`
-          <div class="error">
-            ${this.localize(
+          <ha-alert
+            alert-type="error"
+            .title=${this.localize(
```

Comment:
I don't think we should use title if we don't have a description, I would put the text in the slot.

Reply:
Hm interesting. I was reasoning the other way around. Title is emphasized which is what we want error to be.

Reply:
The element wasn't setup to be used without title, and personally I think it also looks better without the title...